engine/apps/api/views/organization.py

```diff
@@ -66,7 +73,11 @@ def get(self, request):
 
 class GetChannelVerificationCode(APIView):
```

Comment:
I don't believe this view is actually used anymore (I searched through the frontend codebase for `GET current_team/get_channel_verification_code` and didn't come across anything...

Reply:
This one is used in our msteams integration, which is cloud-only (check the frontend custom code in the private repo!).